package.json

```diff
@@ -69,6 +69,7 @@
     "conventional-commit-types": "^2.1.0",
     "copy-webpack-plugin": "^1.1.1",
     "cz-conventional-changelog": "^1.2.0",
+    "d3": "^3.5.17",
```

Comment:
I'm not sure what, if any, v4 features I am using, but I think vega uses v4, and it makes sense to use it since it's been out a while. I'd prefer just doing it now - if it is taking more than a few hours to upgrade we can consider holding off.